CONTRIBUTING.md

```diff
@@ -42,63 +55,66 @@ an issue. Simply choose the issue you would want to work on, and tell everyone
 that you are willing to do so and how you would approach it. The team will be
 happy to guide you and give feedback.
 
-## Coding Guidelines
+## Contributions
 
-We follow the coding guidelines described
-[here](https://github.com/theia-ide/theia/wiki/Coding-Guidelines).
+All contributions to the project must be through pull requests. This applies to all changes to documentation, code, binary files, etc. Even long term committers must use pull requests.
 
-## Sign your work
+No pull request can be merged without being reviewed.
 
-The sign-off is a simple line at the end of the explanation for the patch. Your
-signature certifies that you wrote the patch or otherwise have the right to
-pass it on as an open-source patch. The rules are pretty simple: if you can
-certify the below (from
-[developercertificate.org](http://developercertificate.org/)):
+### Reviewing
 
-```
-Developer Certificate of Origin
-Version 1.1
+For non-trivial contributions, pull requests should sit for at least 24 hours to ensure that committers in other timezones have time to review. Trivial pull requests may be landed after a shorter delay.
 
-Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-1 Letterman Drive
-Suite D4700
-San Francisco, CA, 94129
+The default for each contribution is that it is accepted once no committer has an objection. Once all issues brought by committers are addressed it can be signed-off and landed by any committer.
 
-Everyone is permitted to copy and distribute verbatim copies of this
-license document, but changing it is not allowed.
+### Signing off
 
-Developer's Certificate of Origin 1.1
+Committers sign-off on a pull request by explicitly stating their approval in the PR text or associated comment stream.
```

Comment:
I having a committer sign-off is additional process about responsibility / code ownership, but has nothing to do with copyright. I don't think we need that.

Reply:
> I having a committer sign-off is additional process about responsibility / code ownership, but has nothing to do with copyright.

I am not sure about it. Any sign-off is about copyright. If one signs other unsigned work he declares that he reviewed it and the work is developed under an open-source license to his knowledge. If he is not sure he should not sign but ask the originator.

At node.js they use a PR approval as the sign-off. The DCO does not say that github sign off should be used. The process is very simple a PR should be revied by a committer and approved to be signed.